### PR TITLE
Thenext web

### PR DESCRIPTION
**Pull request by mertcihankurel**
### The Next Web Clone
**İn this project:**
1-) We created media queries.
2-) We updated it for 3 kinds of screen sizes.
3-) Added 2 different navbar for top and bottom.
4-) Used background image for the main section.
5-) Background images cover each fill.
6-) Stickler errors fixed
7-) Html validator issues fixed.
8-) Footer icons have 50% border-radius.

### CODE REVIEW

**Comment by bertil291utn** [comment]
**Status:** Changes requested 🔴  :shipit: ? Not yet

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

Your work is amazing you master it GitHub Flow you have a professional README file. (Check to improve [section](##To_improve))



### Mandatory changes

- [ ] Check the linter files you have stickler config change to [GitHub actions](https://github.com/microverseinc/linters-config/tree/master/html-css)
- [ ] Check your README file live demo section. Fix your markdown syntax


  image

- [ ] Add `overflow-x:hidden` property to avoid horizontal scrolling

##### Small screen
- [ ] Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to seach icon) ([link to page](https://thenextweb.com/))

- [ ] All images take the maximum width and are are aligned in one column ([link to page](https://thenextweb.com/))

- [ ] It aligns 1 articles per row ([link to page](https://thenextweb.com/))

   Each articles has a cover image on the left side and title on the right side of the row ([link to page](https://thenextweb.com/))

|                       ACTUAL WEB PAGE                        |                            YOURS                             |
| :----------------------------------------------------------: | :----------------------------------------------------------: |
| image | image |



##### Medium screen

- [ ] TWN logo appears at the leftmost part ([link to page](https://thenextweb.com/))

- [ ] And also your width page doesn't look great.

image





##### Large screen

- [ ] It contains one bigger image on the leftmost part of the row (around two-thirds of the width) ([link to page](https://thenextweb.com/))
- [ ]  It contains two smaller images on the rightmost part of the row (one above another) ([link to page](https://thenextweb.com/))

image

- [ ] It aligns 4 articles per row ([link to page](https://thenextweb.com/))

  image

 

- [ ] It aligns 3 category boxes per row ([link to page](https://thenextweb.com/))

image

- [ ] It aligns 4 deals per row ([link to page](https://thenextweb.com/))
- [ ] Check your elements is overlapping other ones

image



- [ ] **It does not change through breakpoints** ([link to page](https://thenextweb.com/))

|                       ACTUAL WEB PAGE                        |                            YOURS                             |
| :----------------------------------------------------------: | :----------------------------------------------------------: |
| image | image |

### Summary

Please check again your project when you have a small screen approach.

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Comment by GraceOyiza** [comment]
The errors have been corrected 
1.  Live demo fixed.
2. Corrected overlapping images.
3. All screen sizes fixed.
4. Linters errors fixed.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,842 @@
...
+
+.search-button:hover {
+  color: grey;
+}
+
+@media screen and (max-width: 1300px) {
+  .most {
+    width: 105%;
+  }
+}
```
Comment (bertil291utn) [inline style.css:523]: ## CSS styles should use media queries for 2 breakpoints (screenshot from Pathwright)

- [ ] No small breakpoint as it is mobile-first and the base styles are for the mobile screen. (Screenshot on mobile-first approach from Pathwright)
    - medium: min-width around 768px
    - large: min-width 1024px

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🔴  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Good job on your changes, however, some requests are not changed.

- [ ] You're still forgetting use GitHub Actions to check the linters

### Small screen


Check again [original web page](https://thenextweb.com/) with a small screen and try to compare it with your approach. Work on all changes should be the same as the original one.

- [ ] Navigation and sub-navigation row changes to a hamburger (bars) icon at the rightmost part (next to search icon) ([link to page](https://thenextweb.com/))

- [ ] All images take the maximum width and are are aligned in one column ([link to page](https://thenextweb.com/))

- **Latest news section**

- [ ] It aligns 1 articles per row ([link to page](https://thenextweb.com/))
- [ ] Each articles has a cover image on the left side and title on the right side of the row ([link to page](https://thenextweb.com/))

- **Latest funding section**

- [ ] It aligns 1 articles per row ([link to page](https://thenextweb.com/))

- **Categories (Apps, Gear, Tech, etc) section**

- [ ] It aligns 1 category box per row ([link to page](https://thenextweb.com/))

- **Latest deals section**

- [ ] It aligns 1 deal per row ([link to page](https://thenextweb.com/))



|                       ACTUAL WEB PAGE                        |                        YOUR APPROACH                         |
| :----------------------------------------------------------: | :----------------------------------------------------------: |
| image | image |

### Medium screen


- **Latest Funding section**

- [ ]   It aligns 3 articles per row ([link to page](https://thenextweb.com/))

  image


- **Top 3 articles section**

- [ ] It contains search icon at the rightmost side of sub-navigation row ([link to page](https://thenextweb.com/))

image

- [ ]  It contains TWN logo at the top left corner of the row ([link to page](https://thenextweb.com/))
image




### Footer


- [ ] **It does not change through breakpoints** ([link to page](https://thenextweb.com/))

- [ ] It doesn't contain social media icons vertically and horizontally aligned ([link to page](https://thenextweb.com/))

### General requirements


- [ ] Work on the details you have to approach a clone. You have to on each screening approach (small, medium and large) not only in this section check also other sections

image



### Summary


- Please try to work most on a small screen approach, you have put effort into it.

- You should be the same as the original one, follow all details and check your page don't break any element.

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Comment by GraceOyiza** [comment]
All errors fixed.
Github action created.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,630 @@
...
+                <li><a href="#" class="grey">Terms & Conditions </a></li>
+             </ul>
+                <ul class="left-nav">
+                <li><a href="#" class="grey">About</a></li>
+                <li><a href="#" class="grey">Advertise</a></li>
+                <li><a href="#" class="grey">Jobs</a></li>
+                <li><a href="#" class="grey">Contact</a></li>
+             </ul>
+             </div>          
+         <div class="get">
```
Comment (expjazz) [inline index.html:29]: You have a few problems here. Try to make this more as like the original. I can see that you improved from the last review and we are almost there. Just a few details more.

#### On mobile

| Yours | Original |
|:---:|:---:|
| image | image |

- [ ] The search bar needs to be more close to the left end of the page. 
 

- [ ] There is no hamburger menu

- [ ] The navbar is occupying space from the first article. 

#### Large screen

- [ ] Pay attention to this TNW logo. It should be invisible on the large screen.

| Yours | Original |
|:---:|:---:|
| image | image |

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes requested :recycle: 

### Hello Team. You did a great job :clap: 

However, there are still changes to be done in order to let this project even better. You can check them in the notes below. I would say that generally, the page looks good, but you still need to work on the details, font sizes, and what shows and not shows depending on the screen-sizes.

 Try to make this more like the original. I can see that you improved from the last review and we are almost there. Just a few details more.

#### On mobile

| Yours | Original |
|:---:|:---:|
| image | image |

- [ ] The search bar needs to be more close to the left end of the page. 
 

- [ ] There is no hamburger menu

- [ ] The navbar is occupying space from the first article. 

#### Large screen

- [ ] Pay attention to this TNW logo. It should be invisible on the large screen.


| Yours | Original |
|:---:|:---:|
| image | image |

- [ ] You have linter errors


### Kindly fix them  and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:


_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Comment by GraceOyiza** [comment]
All changes inputted.

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes
## Great job so far team :clap: :100: :+1: you are almost there.

- [ ] please fix the linter errors:
image

### Please make the required changes before you submit another code review.
## happy Coding :tophat:

**Comment by mertcihankurel** [comment]
You can see there must be still stickler errors. But its just a bug. First TSE suggested us change stickler with github action. So we removed stickler file in repo.

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 
## Great job guys you have made an awesome work by applying all the required changes by the previous TSE's.

now you can merge to master your project is approved.

## Happy Coding :tophat:

**Pull request merged**